### ISSUE TITLE
Review token implementaions

### ISSUE THREAD

**Issue by ataker** [department-of-veterans-affairs/vets-design-system-documentation#2190]
## Description

Per [feedback on minimal header implementation](https://github.com/department-of-veterans-affairs/component-library/pull/846#discussion_r1341739250):
> Right now we have a tokens file in the [web-components directory](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/global/_variables.scss), in the [CSS Library](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/css-library/dist/tokens/scss/variables.scss), and in [Formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/9578cd69766b7b75dce415188b73f90c79eb85ea/packages/formation/sass/base/_b-variables.scss#L132). Now we are adding random tokens here in a specific component.
> 
> It just feels like there are tokens everywhere and adding more here might make dealing with them later more challenging than it already will be.

## Tasks
- [ ] We need to review how we are managing our tokens and if we need to reorganize them.
- [ ] Decide if we are allowed to add tokens for individual components.
- [ ] It sounds like we'll be adding more USWDS tokens to our variables file, when will that happen and can some of these decisions wait until then?
- [ ] Do we need a breakpoint tokens file for screen widths?

## Acceptance Criteria
- [ ]

**Comment by humancompanion-usds** [comment]
@micahchiang - Let's get the team on the same page here. I really don't want wanting us already creating tech debt for ourselves to have to clean-up later. We should define the process and then get into a habit of pulling the crank on adding a new token, having that show up in CSS, and being able to access it in a component CSS file (or wherever we need it).

**Comment by micahchiang** [comment]
I created a follow up ticket to address the work needed here - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2236

TLDR: we are consolidating css variables into css library. 

Will close this for now, since the tasks were for meeting and discussing a way forward.
